app/assets/stylesheets/non-main.scss

```diff
@@ -1,7 +1,11 @@
 /*left side area styles*/
 .left-content {
-	margin-top: 6rem;
+	padding-top: 4.5rem;
```

Comment:
The flash messages to show up incorrectly, and I think it's this line that's doing it.
![screen shot 2015-01-20 at 1 57 43 pm](https://cloud.githubusercontent.com/assets/2523812/5827093/984009cc-a0ac-11e4-8a93-95591f029d96.png)